docs/docs/recipes/organizations/integration.mdx

```diff
@@ -0,0 +1,276 @@
+```
+
+The `organizations` field (claim) will also be included in response from the [UserInfo endpoint](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo).
+
+### Get organization roles
+
+To get all organization roles of the current user:
+
+- If you are using a Logto SDK with Organizations support, you can add the `urn:logto:scope:organization_roles` scope to `scopes` parameter of the configuration object. Usually the SDK will have an enum for this scope, e.g. `UserScope.OrganizationRoles` in [Logto JS SDKs](https://github.com/logto-io/js).
+- For other cases, you need to add the `urn:logto:scope:organization_roles` scope to the `scope` parameter of the SDK config (or auth request).
+
+Then you can get the organization roles from the `idToken`:
+
+```ts
+// Use JavaScript as an example
+const idToken = await logto.getIdTokenClaims();
+
+console.log(idToken.organization_roles); // A string array of organization roles
+```
+
+Each string in the array is in the format of `organization_id:role_id`, e.g. `org_123:admin` means the user has the `admin` role in the organization with ID `org_123`.
+
+The `organization_roles` field (claim) will also be included in response from the [UserInfo endpoint](https://openid.net/specs/openid-connect-core-1_0.html#UserInfo).
+
+## Fetch access token for an organization
+
+To perform actions in the context of an organization, the user needs to be granted an access token for that organization (organization token). The organization token is a JWT token that contains the organization ID and the user's permissions (scopes) in the organization.
+
+### Step 1: Add parameters to the authentication request
+
+- If you are using a Logto SDK with Organizations support, you can add the `urn:logto:scope:organization_token` scope to `scopes` parameter of the configuration object, the same way as [Get organization IDs of the current user](#get-organization-ids-of-the-current-user).
+  - Logto SDK with Organizations support will automatically handle the rest of the configuration.
+- For other cases, you need to add the `offline_access` and `urn:logto:scope:organizations` scopes to the `scope` parameter and the `urn:logto:resource:organizations` resource to the `resource` parameter of the SDK config (or auth request).
+  - Note: `offline_access` is required to get the `refresh_token` that can be used to fetch organization tokens.
+
+<Tabs groupId="sdk">
+
+  <TabItem value="js" label="JavaScript">
+
+```ts
+const logto = new LogtoClient({
+  // ...
+  scopes: [UserScope.Organizations],
+});
+```
+
+  </TabItem>
+  <TabItem value="react" label="React">
+
+```tsx
+<LogtoProvider
+  config={{
+    // ...
+    scopes: [UserScope.Organizations],
+  }}
+></LogtoProvider>
+```
+
+  </TabItem>
+
+  <TabItem value="others" label="Others">
+
+```ts
+const config = {
+  // ...
+  scope: 'openid offline_access urn:logto:scope:organizations',
+  resource: 'urn:logto:resource:organizations',
```

Comment:
When using SDKs, the `resource` will be added automatically? Maybe we should point it out.

Reply:
<img width="555" alt="image" src="https://github.com/logto-io/docs/assets/14722250/dd3995b0-79ae-4164-be9d-3ea8d9eb3e8a">

this?